packages/vite/src/node/server/pluginContainer.ts

```diff
@@ -306,6 +310,28 @@ export async function createPluginContainer(
       return out as ResolvedId | null
     }
 
+    async load(
+      options: {
+        id: string
+        resolveDependencies?: boolean
+      } & Partial<PartialNull<ModuleOptions>>,
+    ): Promise<ModuleInfo> {
+      // We may not have added this to our module graph yet, so ensure it exists
+      await moduleGraph?.ensureEntryFromUrl(options.id)
+      // Not all options passed to this function make sense in the context of loading individual files,
+      // but we can at least update the module info properties we support
+      updateModuleInfo(options.id, options)
+
+      await container.load(options.id)
```

Comment:
```suggestion
      await container.load(options.id, { ssr: this.ssr })
```
I think we should pass `ssr` flag.

Reply:
Yeah that makes sense